norwegian/README.md

```diff
@@ -69,13 +73,12 @@ There's no headlines.
 
 **NOTE: there's a newer table from 2011 available; see above.**
 
-[Norsk Punktskrift - del 4](1995%20-%20Norsk%20Punktskrift%20-%20del%204.pdf)
------------------------------------------------------------------
+## [Norsk Punktskrift - del 4](1995%20-%20Norsk%20Punktskrift%20-%20del%204.pdf)
 
  * _Title translated_: Norwegian Braille - part 4
  * _by_: [Huseby kompetansesenter - Statlig spesialpedagogisk senter for synshemmede](http://www.statped.no/Spraksider/In-English/)
  * _published_: Oslo, 1995
  * _language_: Norwegian
  * _isbn_: 82-7740-002-0
 
-[Table of contents](1995%20-%20Norsk%20Punktskrift%20-%20del%204.toc.md)
+## [Table of contents](1995%20-%20Norsk%20Punktskrift%20-%20del%204.toc.md)
```

Comment:
Is this a mistake?

Reply:
Whops, yeah, good catch. I'll commit a fix...